monitor/cli.py

```diff
@@ -106,12 +112,15 @@ def monitor(
     profiles_dir,
     update_dbt_package,
     full_refresh_dbt_package,
-    profile_target
+    profile_target,
+    dbt_vars
 ):
     click.echo(f"Any feedback and suggestions are welcomed! join our community here - "
                f"https://bit.ly/slack-elementary\n")
     if ctx.invoked_subcommand is not None:
         return
+    if dbt_vars is not None:
+        vars = ordered_yaml.loads(dbt_vars)
```

Comment:
When `dbt_vars` is not provided, then `vars` is undefined and calling ```data_monitoring.run(days_back, full_refresh_dbt_package, vars=vars)``` will throw an exception.
To prevent that you can use an inline if statement:
```python
vars = ordered_yaml.loads(dbt_vars) if dbt_vars else None
```

Reply:
fixed